requirements/test.in

```diff
@@ -1,3 +1,4 @@
 flaky
```

Comment:
initially, I posted without these deps, but after some tests started failing I remembered that `unittest.mock` is relatively new.  This should fix the tests.